backend/transcribee_backend/config.py

```diff
@@ -69,7 +71,10 @@ def get_short_page_config() -> Dict[str, ShortPageConfig]:
 
 
 def get_public_config():
-    return PublicConfig(models=get_model_config())
+    return PublicConfig(
+        models=get_model_config(),
+        logged_out_redirect_url=environ.get("TRANSCRIBEE_LOGGED_OUT_REDIRECT_URL"),
```

Comment:
Please pull this from the `Setttings` instance, which gives more flexible ways of setting this (for example via a .env file)

Reply:
Aaah I did not understand that, sounds good :)